laravel/socialite/2015-07-23.yaml

```diff
@@ -0,0 +1,11 @@
+title:     Insecure state generation
+link:      https://github.com/laravel/socialite/pull/91
```

Comment:
btw, the explanation given in https://github.com/laravel/socialite/pull/93#discussion_r36254365 is precisely related to the fact that the actual issues and the action taken are not described